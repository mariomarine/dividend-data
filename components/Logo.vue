<template>
	<div class="Logo">
		<h1>Dividend Data</h1>
	</div>
</template>

<style>
</style>

