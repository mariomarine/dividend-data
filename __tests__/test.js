import { mount } from '@vue/test-utils';
import Logo from '../components/Logo.vue';

describe('App.vue', () => {
  test('should mount for testing', () => {
    expect(1).toEqual(1);
  });
  test('should render content correctly', () => {
	const wrapper = mount(Logo);
	expect(wrapper.vm.$el).toMatchSnapshot();
  });
});

