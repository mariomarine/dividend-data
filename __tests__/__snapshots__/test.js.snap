// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue should render content correctly 1`] = `
<div
  class="Logo"
>
  <h1>
    Dividend Data
  </h1>
</div>
`;
